src/tests/integration/test-data/foundry/test/FreshInt.t.sol

```diff
@@ -30,4 +33,32 @@ contract FreshIntTest is Test, KEVMCheats {
         int128 val = int128(uint128(kevm.freshUInt(16)));
         assertGt(val, max);
     }
+
```

Comment:
I think that these tests belong in a different file. `FreshInt.t.sol` made sense for tests about symbolic integers, and borderline symbolic Booleans, as they are treated as integers, but I do not think that symbolic bytes fit the description.

Reply:
Fixed.